core/new-gui/src/app/dashboard/user/component/sort-button/sort-button.component.ts

```diff
@@ -20,10 +21,12 @@ export class SortButtonComponent {
     const update = () => {
       this._entries = value;
       this.entriesChange.emit(value);
+      this.sort();
```

Comment:
`this.entriesChange.emit(value);` should go before sort. I can push a change for this

Reply:
I mean sort before emit

Reply:
fixed.